handsontable/test/e2e/settings/fixedColumnsStart.spec.js

```diff
@@ -0,0 +1,452 @@
+        fixedColumnsLeft: 3
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(3);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 2),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(2);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 1),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(1);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 0),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(0);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 1),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(1);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 2),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(2);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 3),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(3);
+
+      updateSettings({
+        data: Handsontable.helper.createSpreadsheetData(3, 4),
+      });
+
+      expect(getLeftClone().find('tbody tr:eq(0) td').length).toBe(3);
+    });
+  });
+  describe('fixedColumnsLeft with RTL', () => {
+    const id = 'testContainer';
+
+    beforeEach(function() {
+      this.$container = $(`<div dir="rtl" id="${id}"></div>`).appendTo('body');
+    });
+
+    afterEach(function() {
+      if (this.$container) {
+        destroy();
+        this.$container.remove();
+      }
+    });
+
+    it('constructor should throw Error', () => { // todo causes another tests to crash
+      expect(() => handsontable({
+        fixedColumnsLeft: 3
+      })).toThrow();
+    });
+
+    it('updateSettings should throw Error', () => { // todo causes another tests to crash
+      handsontable();
+
+      expect(() => updateSettings({
+        fixedColumnsLeft: 4
+      })).toThrow();
+    });
+  });
+  describe('fixedColumnsLeft with fixedColumnsStart', () => { // todo should throw errors
+    const id = 'testContainer';
+
+    beforeEach(function() {
+      this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+    });
+
+    afterEach(function() {
+      if (this.$container) {
+        destroy();
+        this.$container.remove();
+      }
+    });
+
+    it('defined both in constructor should thrown an error', () => {
+      expect(() => handsontable(
+        {
+          fixedColumnsStart: 3,
+          fixedColumnsLeft: 1
+        }
+      )).toThrow();
+
+    });
+
+    it('defined `fixedColumnsLeft` in constructor, `fixedColumnsStart` in updateSettings should thrown an error', () => {
+      handsontable({
+        fixedColumnsLeft: 2
+      });
+
+      expect(() => updateSettings({
+        fixedColumnsStart: 4
+      }));
+
```

Comment:
```suggestion
      })).toThrowError('The `fixedColumnsLeft` and `fixedColumnsStart` should not be used together. Please use only the option `fixedColumnsStart`.');
```
Missing expectation :)